installer/functions/plugin/get-plugins.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import { convertToBase64, listPluginsEndpoint } from './helper-plugins.private';
+import axios from 'axios';
+import { getParam, ACCOUNT_SID, AUTH_TOKEN } from '../common';
+
+/**
+ * This function is used to get all current plugins.
+ * If a plugin is not deployed via Docker, it will not be listed.
+ * Returns a list of plugins that have been deployed on the current account 
+ * provided we give it the SID and Auth Token
+ *  */ 
+export const handler: ServerlessFunctionSignature = async function(
+  context: Context,
+  event: {},
+  callback: ServerlessCallback
+) {
+
+  const response = new Twilio.Response();
+  response.appendHeader('Access-Control-Allow-Origin', '*');
+  response.setStatusCode(200);
+
+  try {
+    const accountSid = await getParam(context, ACCOUNT_SID);
+    const authToken = await getParam(context, AUTH_TOKEN);
```

Comment:
this could proobably be optimized... the above two sequential calls will fetch 20 accounts each and use just the first one in the list for both. 

one way i could see this being optimized is to create that `getActiveAccount` to fetch a single `account`, and then just access `.sid` and `.authToken` on the object